.github/workflows/ci.yml

```diff
@@ -1,57 +1,45 @@
-name: ExceptionPage CI
+name: CI
 
 on:
+  pull_request:
   push:
     branches: [master]
-  pull_request:
-    branches: "*"
+  schedule:
+    - cron: "0 3 * * 1" # Every monday at 3 AM
 
 jobs:
-  check_format:
+  test:
     strategy:
       fail-fast: false
       matrix:
-        crystal_version:
-          - 0.36.1
-          - 1.0.0
-        experimental:
-          - false
-    runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.experimental }}
-    container: crystallang/crystal:${{ matrix.crystal_version }}
-    steps:
-      - uses: actions/checkout@v2.3.4
-      - name: Install shards
-        run: shards install
-      - name: Format
-        run: crystal tool format --check
-      - name: Lint
-        run: ./bin/ameba
+        os: [ubuntu-latest]
+        crystal: [latest, nightly]
```

Comment:
I feel like 0.36.1, and 1.0.0 should be specified because many people won't be able to upgrade to 1.0.0 for a bit as shards are still being updated. My thought was this shard would support the previous release, and the current release, then do an experimental nightly so that we can catch if something changes, but also disregard failures if we needed.

What are your thoughts?

Reply:
Yup, I was thinking the same - previous, current (`latest`) and the next (`nightly`). I was wondering tough, whether there could be added a fluent version selector to be used already as a part of [`install-crystal`](https://github.com/oprypin/install-crystal) action, probably sth like `latest - n`, or at least `previous`.

/cc @oprypin WDYT?

Reply:
Ah, that would be kinda cool. Since Crystal releases are pretty spread out, it's not like we'd see this sort of deal breaking super often, so being a little more fluid in it might be ok. I figure when 1.1.0 drops, most shards will slowly start to drop 0.36 support anyway.

Reply:
For the time being I've added 0.36.1 to the list, so this should be GTG.

Reply:
I think "previous version" would be possible to add. Not sure when I'll do it.

And at the same time, I probably would recommend to instead support an explicit old version, so that you can notice when you're breaking the promise in `shard.yml` regarding supported Crystal version, and then update the version in both places at the same time.